tools/topology/topology2/doc/README

```diff
@@ -0,0 +1,7 @@
+To build the topology2 source documentation do following steps:
+
+cd tools/topology/topology2/doc
+cmake -B build
+cmake --build build
```

Comment:
Since everything (--options, actions, scripts, tools, failures, CI, verbs, nouns, etc.) is always called the same "build", adding just one trailing slash character makes it obvious that it is a directory in this case.
```suggestion
cmake -B build/
cmake --build build/ -v 
```

This cmake file is not verbose at all (cause it does  nothing, just calls a shell script) so might as well use `-v`.